tests/js/client/communication/test-communication.js

```diff
@@ -309,19 +310,20 @@ function GenericAqlSetupPathSuite(type) {
       reconnectRetry(endpoint, db._name(), "root", "");
       
       let res = arango.GET_RAW('/_admin/debug/failat');
+      reconnectRetry(primaryEndpoint, "_system", "root", "");
       return res.code === 200;
     } catch(ex) {
       reconnectRetry(primaryEndpoint, "_system", "root", "");
       throw (ex);
     }
-    return false;
   }
 
   /// @brief set failure point
   function debugSetFailAt(endpoint, failAt) {
     try {
       reconnectRetry(endpoint, db._name(), "root", "");
       let res = arango.PUT_RAW('/_admin/debug/failat/' + failAt, {});
+      reconnectRetry(primaryEndpoint, "_system", "root", "");
```

Comment:
This will now reconnect twice in case of error.
Please replace this by a simple `finally` (see above)
Unfortunately GitHub does not allow me to da a multiline comment here